### PR TITLE
Adding Distribution creation and Documentation for Connector

### PR DESCRIPTION
**Pull request by Ashi1993**
## Pull Request Title

> Explain  in a few lines the purpose of this pull request

**Issue link:** *required*

------

### Development Checklist

1. [ ] Built complete solution with pull request in place.
2. [ ] Ran checkstyle plugin with pull request in place.
3. [ ] Ran Findbugs plugin with pull request in place.
4. [ ] Formatted code according to WSO2 code style.

### Secure Development Checklist

1. [ ] Ran FindSecurityBugs plugin and verified report.
2. [ ] Ran Dependency-check plugin and verified report for new dependencies added.
2. [ ] Ran Dependency-check plugin and verified report for dependency version changes.
3. [ ] Have you verify the PR does't commit any keys, passwords, tokens, usernames, or other secrets?
4. [ ] Have you followed secure coding standards in [WSO2 Secure Engineering Guidelines](http://wso2.com/technical-reports/wso2-secure-engineering-guidelines)?

### Testing Checklist

1. [ ] Written unit tests.
2. [ ] Documented test scenarios(link available in guides).
3. [ ] Written automation tests (link available in guides).

**Automation Test Details**

| Test Suite        | Test Script IDs   |
| ----------------- | ----------------- |
| Integration Suite | *TCXXXXX, TCXXXX* |

### CODE REVIEW

File: distribution/resources/bfsi-consent-management.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                <ClientId>0NXYPMnh9mxSPCVjvvsVh17LWDca</ClientId>
+                <ClientSecret>pfVkfiEUvQqmZdYQuruauOs6SS0wd7yggkR2of58NXsa</ClientSecret>
+            </ClientCredentials>
+            <Params>
+                <IdentityServerBaseUrl>https://localhost:9443</IdentityServerBaseUrl>
+                <ApiManagerServerBaseUrl>https://localhost:8243</ApiManagerServerBaseUrl>
+            </Params>
+        </Portal>
+    </Consent>
+</Server>
```
Comment (aka4rKO) [inline distribution/resources/bfsi-consent-management.xml]: add new line

File: distribution/src/assembly/bin.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!--
+ ~ Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```
Comment (aka4rKO) [inline distribution/src/assembly/bin.xml]: lets change this to 2024

File: docs/resources/mysql.sql [hunk 3]
```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```
Comment (aka4rKO) [inline docs/resources/mysql.sql]: lets change this to 2024

Comment (Ashi1993) [reply docs/resources/mysql.sql]: Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/11/commits/a00b9b419db2aa1ba4198ff1a8f77e6bd545b1c4

File: distribution/src/assembly/bin.xml (hunk 2 above)
Comment (Ashi1993) [reply distribution/src/assembly/bin.xml]: Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/11/commits/a00b9b419db2aa1ba4198ff1a8f77e6bd545b1c4

File: distribution/resources/bfsi-consent-management.xml (hunk 1 above)
Comment (Ashi1993) [reply distribution/resources/bfsi-consent-management.xml]: Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/11/commits/a00b9b419db2aa1ba4198ff1a8f77e6bd545b1c4

**Review by aka4rKO — DISMISSED** [review]

File: docs/config.md [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+
+1. Add the root and issuer of the client certificate to the client trustore.
+``` bash
+keytool -import -alias <alias_name> -file <localtion_to-the-root-certificate> -keystore <IS_HOME>/repository/resources/security/client-truststore.jks -storepass wso2carbon
+```
+``` bash
+keytool -import -alias <alias_name> -file <localtion_to-the-issuer-certificate> -keystore <IS_HOME>/repository/resources/security/client-truststore.jks -storepass wso2carbon
+```
+
+2. Add the following configurations to the deployment toml file
```
Comment (VimukthiRajapaksha) [inline docs/config.md]: Shall we mention the relative path of the toml file?

File: docs/config.md [hunk 5]
```diff
@@ -0,0 +1,547 @@
...
+
+- Add the following data source configuration
+
+``` toml tab="MySQL"
+[[datasource]]
+id="WSO2BFSI_DB"
+url = "jdbc:mysql://localhost:3306/bfsi_consentdb?autoReconnect=true&amp;useSSL=false"
+username = "root"
+password = "root"
```
Comment (VimukthiRajapaksha) [inline docs/config.md]:
```suggestion
url = "jdbc:mysql://<hostname:port>/bfsi_consentdb?autoReconnect=true&amp;useSSL=false"
username = "<username>"
password = "<password>"
```

**Review by VimukthiRajapaksha — DISMISSED** [review]

File: docs/config.md (hunk 5 above)
Comment (Ashi1993) [reply docs/config.md]: Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/11/commits/d9302fc7f954430ce59b548ec2243eb849ba3330

File: docs/config.md (hunk 4 above)
Comment (Ashi1993) [reply docs/config.md]: Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/11/commits/d9302fc7f954430ce59b548ec2243eb849ba3330

**Review by VimukthiRajapaksha — APPROVED** [review]

**Review by aka4rKO — APPROVED** [review]

**Pull request merged**